packages/upstream/ssl.0.5.9/opam

```diff
@@ -9,16 +9,16 @@ build: [
 ]
 depends: [
   "ocaml" {>= "4.02.0"}
-  "dune" {build & >= "1.2.1"}
+  "dune" {>= "1.2.1"}
   "base-unix"
   "conf-openssl"
 ]
 synopsis: "Bindings for OpenSSL"
 authors: "Samuel Mimram <samuel.mimram@ens-lyon.org>"
 url {
-  src: "https://github.com/savonet/ocaml-ssl/archive/0.5.7.tar.gz"
+  src: "https://github.com/savonet/ocaml-ssl/archive/0.5.9.tar.gz"
```

Comment:
Lets defer this `ssl` update for now: it enables certificate checking by default which require updating a lot of tests and code on our side to deal with it.
We should update everything else, and leave this one last as a separate PR, branch.